packages/xo-web/src/common/selectors.js

```diff
@@ -550,29 +555,41 @@ export const createGetVmDisks = vmSelector =>
     )
   )
 
-export const createGetLoneSnapshots = createGetObjectsOfType(
-  'VM-snapshot'
-).filter(
-  create(
-    _createCollectionWrapper(
-      (_, props) => props.schedules !== undefined && map(props.schedules, 'id')
-    ),
-    scheduleIds =>
-      scheduleIds
-        ? _ => {
-            const scheduleId = _.other['xo:backup:schedule']
-            return scheduleId !== undefined && !scheduleIds.includes(scheduleId)
-          }
-        : false
-  )
-)
-
 export const getIsPoolAdmin = create(
   create(createGetObjectsOfType('pool'), _createCollectionWrapper(Object.keys)),
   getCheckPermissions,
   (poolsIds, check) => some(poolsIds, poolId => check(poolId, 'administrate'))
 )
 
+export const getLoneSnapshots = create(
+  create(createGetObjectsOfType('VM-snapshot'), snapshots => {
+    const backupSnapshots = []
+    forEach(snapshots, snapshot => {
+      const other = snapshot.other
+      const jobId = other['xo:backup:job']
+      if (jobId !== undefined) {
+        backupSnapshots.push({
+          ...snapshot,
+          jobId,
+          vmId: other['xo:backup:vm'],
+          scheduleId: other['xo:backup:schedule'],
+        })
+      }
+    })
+    return backupSnapshots
+  }),
```

Comment:
Related to [this comment](https://github.com/vatesfr/xen-orchestra/pull/5125#discussion_r452275813), you can add a `createFilter` layer to get the snapshots that have a `xo:backup:job` entry so that it doesn't trigger a recompute if an unrelated snapshot changed.